### ISSUE TITLE
Installation failed on Fedora Silverblue (ostree-based)

### ISSUE THREAD

**Issue by CheariX** [safing/portmaster#733]
**What did not work?**

When I try to install portmaster on Fedora Silverblue 36 with `rpm-ostree install portmaster-installer.rpm`., the operation fails in ostree's `%post` phase:

```
journalctl -t 'rpm-ostree(portmaster.post)'
> rpm-ostree(portmaster.post)[67337]: /proc/self/fd/5: line 30: [: : integer expression expected
> rpm-ostree(portmaster.post)[67337]: /proc/self/fd/5: line 36: [: : integer expression expected
> rpm-ostree(portmaster.post)[67337]: portmaster: Configuring portmaster.service to launch at boot
> rpm-ostree(portmaster.post)[67352]: rpm-ostree-systemctl: Ignored non-preset command: enable portmaster.service
> rpm-ostree(portmaster.post)[67353]: chcon: failed to change context of '/opt/safing/portmaster/portmaster-start' to ‘system_u:object_r:bin_t:s0’: Operation not supported
```

I'm not really sure what the issue is.  Seems to be ralted to 1) SELinux, 2) OSTree, and 3) installation in `/opt`. 
On Fedora Workstation 36, portmaster's installation works like charm.

**Debug Information**:

Fedora Silverblue 36.20220708.0

**Comment by p2anav** [comment]
I think it's related to SELinux. Follow the docs - https://docs.safing.io/portmaster/install/linux#security-enhanced-linux-selinux

**Comment by CheariX** [comment]
Thank you. I think you are right: the reason could be SELinux.

> This happens because SELinux will not allow you to run a binary from /opt/safing/portmaster as systemd service.

Unfortunately, the docs assume that portmaster was installed and then advice to change SELinux policy afterwards.
On Silverblue, the installation fails with the error stated above. So I cannot change anything afterwards.

I tried it on Fedora Workstation 36. No issues occured and I did not need to change any SELinux settings.
